### ISSUE TITLE
Hilgrund's Tomb

### ISSUE THREAD

**Issue by Untiltedpig** [SkyrimTogether/issues-game#354]
## Description

How to reproduce this issue:

1. Enter Hilgrund's Tomb with multiple people
2. Intense fps lag for all players.

## Reproduction rate

How many times can you reproduce it?
100% of the time
## Expected result
Intense Lag

## Your environment

    * The Elder Scrolls V: Skyrim Special Edition
    
    * Windows 10 x64

**Comment by skreborn** [comment]
Are you sure your expected result is "intense lag"? I feel like that's a mistake.

**Comment by Untiltedpig** [comment]
You're right I meant to write that you would expect no intense lag, I was very tired when I wrote this sorry about that.

**Comment by theo-ardouin** [comment]
Experienced it too. Generally speaking, it seems that everytime there are headstones to turn, there is a drop of FPS in said area.
